### ISSUE TITLE
Mesh duplicates with Blender 3.1.2, RenderDoc v1.19, and Importer v0.5.0

### ISSUE THREAD

**Issue by dan52896** [eliemichel/MapsModelsImporter#221]
Hello! My captures from RenderDoc 1.19 seem to have two of each mesh. For example, Mesh-00000 and Mesh-00001 are the same, and Mesh-00002 and Mesh-00003 are the same, and so on. Any ideas of what's causing this? Thanks!

**Comment by EODEFX** [comment]
I'm having the same issue. This doesn't fix the bug, but the 2.8 version of the answer given on this page: [Stack Exchange](https://blender.stackexchange.com/questions/87489/cleaning-unwanted-overlapping-duplicate-meshes), seems to work. I ran the script and just deleted the objects that it selected and I didn't notice any holes or anything left behind.

Code if something happens to that link:
```
import bpy
from mathutils import Vector
context = bpy.context

scene = context.scene
view_layer = context.view_layer
TOL = 1.0e-3 # reduced tolerance.

# routine to text matches on bound box.
def test(ob1, ob2):
    # test if ob1 and ob2 which have same face/edge/vert count are same
    bbox1 = [ob1.matrix_world @ Vector(b) for b in ob1.bound_box]
    bbox2 = [ob2.matrix_world @ Vector(b) for b in ob2.bound_box]

    diffs = [v1 - v2 for  v1, v2 in zip(bbox1, bbox2)]    
    return all(d.length < TOL for d in diffs)
    #return sum(diffs, Vector()).length < TOL

obs = {}
keys = []
for ob in context.selected_objects:
    ob.select_set(False)
    if ob.type != 'MESH':
        continue
    # make a key from faces, edges, verts
    mesh = ob.data
    key = "%d_%d_%d" % (len(mesh.polygons), len(mesh.edges), len(mesh.vertices))
    obs[ob.name] = key
    keys.append(key)

possible_dupes = set([x for x in keys if keys.count(x) > 1])
print("-" * 50)
while possible_dupes:
    p = possible_dupes.pop()
    dupes = sorted(name for name, value in obs.items() if value == p)
    while dupes:
        original_ob = scene.objects.get(dupes.pop(0))
        for dupe in dupes:

            dupe_ob = scene.objects.get(dupe)
            if dupe_ob.select_get():
                continue

            if test(original_ob, dupe_ob):
                dupe_ob.select_set(True)
                view_layer.objects.active = dupe_ob
                print("possible dupe: ", dupe, " of original: ", original_ob.name)
```

**Comment by Gordon1fm** [comment]
same here, Had to write script to remove, every uneven number. Blender 3.2, MapsModelsImporter-v0.5.0, RenderDoc v1.19,

**Comment by eliemichel** [comment]
This has been fixed by ddb4523227c6c2b89dc1cd14a8d4331650b6c091
